Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -53,6 +56,30 @@ class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > :
                let jsonData = try? JSONSerialization.data(withJSONObject: object, options: [.prettyPrinted]) {
                 outputText = String(data: jsonData, encoding: .utf8)!
             }
+        case .gpx:
+            var gpxText: String = String("<?xml version=\"1.0\" encoding=\"UTF-8\"?>")
+            gpxText.append("\n<gpx xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns=\"http://www.topografix.com/GPX/1/1\" xsi:schemaLocation=\"http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd\" version=\"1.1\">")
+            
+            guard let routeResponse = routeResponse,
+                  let routes = routeResponse.routes else { return }
+            
+            let timeInterval: TimeInterval = 1
+            let dateFormatter = ISO8601DateFormatter()
+            dateFormatter.formatOptions = .withInternetDateTime
+            var time = Date()
+            
+            routes.forEach { route in
```

Comment:
Each route in the response will appear in order in the GPX, causing it to jump from the arrival step of one route to the departure step of another route. This is reasonable for simulation purposes: if the user only wants one route, they can request no alternatives.